### DISCUSSION TITLE
STM32Cube Ide vs VS code

### DISCUSSION THREAD

**Discussion by yuthnea** [bitcraze/discussions#1540] [General]
Hi,
could i use STM32Cube ide for implement Firmware.

**Comment by ataffanel** [comment]
I have never used STM32Cube , however since it is an Eclipse-based IDE I am sure it is possible to bend it to work with the Crazyflie firmware project. Though, most of the advanced functionality implemented by ST will likely not work. Crazyflie firmware is a Makefile-based project using and this is something Eclipse should handle.
